compression.go

```diff
@@ -14,104 +12,15 @@ type CompressionType int
 const (
 	CompressionTypeNone   CompressionType = 0
 	CompressionTypeSnappy CompressionType = 1
+	CompressionTypeZstd   CompressionType = 2
 )
 
-var _ straw.StreamStore = &snappyStreamStore{}
-
 func newSnappyStreamStore(store straw.StreamStore) *snappyStreamStore {
 	return &snappyStreamStore{store}
 }
 
-// snappyStreamStore is a straw.StreamStore wrapper that implements transparent snappy compression. Everything is supported, except for calling Size() on the os.FileInfo returned from Stat or Lstat.  Calling Size() like this will panic, but freezer does not need that functionality anyway.
-type snappyStreamStore struct {
```

Comment:
I guess you wanted to move this into a new file, but that doesn't seem to be anywhere...